runtime-jvm/main/src/main/scala/Builtins.scala

```diff
@@ -11,33 +11,78 @@ import org.unisonweb.util.{Sequence, Stream, Text}
 
 /* Sketch of convenience functions for constructing builtin functions. */
 object Builtins {
+  def env =
+    (new Array[U](20), new Array[B](20), StackPtr.empty, Result())
 
   // Stream.empty : Stream a
   val Stream_empty =
-    c0l("Stream.empty", Stream.empty[Value])
+    c0z("Stream.empty", Stream.empty[Value])
 
 
-  // Stream.fromInt : Int64 -> Stream Int64
-  val Stream_fromInt = // Stream.iterate(unison 0)(Int64_inc)
-    fp_l("Stream.fromInt", "n", Stream.fromUnison)
+  // Stream.fromInt : Integer -> Stream Integer
+  val Stream_fromInt = // Stream.iterate(unison 0)(Integer_inc)
+    fp_z("Stream.fromInt", "n", Stream.fromUnison)
```

Comment:
just `Stream.from`, or even define this function in Unison as `Stream.iterate 0 (+)`